ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -154,29 +154,13 @@ let send_diagnostics ?diagnostics rpc doc =
           Server.notification rpc notif))
 
 let on_initialize rpc (ip : Lsp.Types.InitializeParams.t) =
-  let log_consumer (section, title, text) =
-    if title <> Logger.Title.LocalDebug then
-      let type_, text =
-        match title with
-        | Error -> (MessageType.Error, text)
-        | Warning -> (Warning, text)
-        | Info -> (Info, text)
-        | Debug -> (Log, Printf.sprintf "debug: %s" text)
-        | Notify -> (Log, Printf.sprintf "notify: %s" text)
-        | Custom s -> (Log, Printf.sprintf "%s: %s" s text)
-        | LocalDebug -> failwith "impossible"
-      in
-      let message = Printf.sprintf "[%s] %s" section text in
-      let notif = Server_notification.LogMessage { message; type_ } in
-      let (_ : _ Fiber.t) =
-        let state : State.t = Server.state rpc in
-        task_if_running state ~f:(fun () -> Server.notification rpc notif)
```

Comment:
By the way, this code was just broken.  It was ignoring a fiber value so I don't know how this was ever logged.